operator/src/main/java/org/keycloak/operator/crds/v2alpha1/realmimport/KeycloakRealmImportSpec.java

```diff
@@ -45,4 +48,11 @@ public void setRealm(RealmRepresentation realm) {
         this.realm = realm;
     }
 
+    public Boolean isReplacePlaceholders() {
```

Comment:
```suggestion
    public Boolean getReplacePlaceholders() {
```

Thanks for the update - sorry I was out the latter part of the week and may have missed the change.